Ginger/GingerCoreNET/RunLib/CLILib/CLIProcessor.cs

```diff
@@ -423,33 +433,43 @@ void ExecuteRunSet()
                 Environment.ExitCode = 1; //failure
             }
 
-            Reporter.ToLog(eLogLevel.DEBUG, "Closing Solution and doing Cleanup...");
+            Reporter.ToLog(eLogLevel.INFO, "Closing Solution and doing Cleanup...");
             mCLIHelper.CloseSolution();            
         }
 
-        private void CLILoadAndPrepare(string runsetConfigs="")
+        private bool CLILoadAndPrepare(string runsetConfigs="")
         {
-            if (!mCLIHelper.LoadSolution())
+            try
             {
-                return;//failed to load Solution;
-            }
+                if (!mCLIHelper.LoadSolution())
+                {
+                    return false;//failed to load Solution;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4886178)